frontend/src/static/css/style.scss

```diff
@@ -115,7 +116,7 @@ header {
     position: fixed;
     right: $tabs-width;
     width: 2px;
-    z-index: 100;
+    z-index: map-get($z-indices, 'center-divider');
```

Comment:
I think u forgot to use `z-index('centre-divider')` here

Reply:
Sorry, yes I missed this. Fixed.